src/eduid_idp/login.py

```diff
@@ -300,67 +285,29 @@ def _get_login_response_authn(self, ticket, user):
         req_authn_context = self._get_requested_authn_context(ticket)
 
         # XXX loop though the list of all authn_context_class_ref!
-        auth_levels = self._get_acceptable_auth_levels(req_authn_context)
-        try:
-            response_authn = eduid_idp.assurance.response_authn(req_authn_context, session_authn, auth_levels,
-                                                                self.logger)
-        except eduid_idp.error.Forbidden:
-            # The level of authentication was not sufficient for the requested AuthnContext.
-            raise MustAuthenticate()
+        #auth_levels = self._get_acceptable_auth_levels(req_authn_context)
+        #try:
+        #    response_authn = eduid_idp.assurance.response_authn(req_authn_context, session_authn, auth_levels,
+        #                                                        self.logger)
+        #except eduid_idp.error.Forbidden:
+        #    # The level of authentication was not sufficient for the requested AuthnContext.
+        #    raise MustAuthenticate()
```

Comment:
Do you keep these commented lines for a reason?

Reply:
They served the purpose of reminding me to raise MustAuthenticate() again after the refactor - which I've now added another commit to do =)